src/transformStateWithClones.js

```diff
@@ -7,7 +7,48 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const objectLog = [];
+  const objectToModify = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        const dataToAdd = action.extraData;
+
+        addProperty(objectToModify, dataToAdd);
+        break;
+
+      case 'removeProperties':
```

Comment:
you can set these strings to variables, and use variables as a property

Reply:
➕